05_syntax_lab/07.py

```diff
@@ -1,8 +1,15 @@
-""" Write a program that selects a random number
-and asks the user to guess it.
-
-After each guess print a hint "too large" or "too small" to the user.
-
-Bonus: To make things interesting, the program should cheat once in a white
-"""
-
+################# Part 7
+while 1:
+    r1 = randint(1, 10)
+    #print('{}'.format(r1))
+    print('Please gusee a number in the range 1 - 100')
+    r = int(input())
+    if r == r1:
+        print('BINGO')
+        break
+    if r < r1:
+        print('your guess ({}) is too small'.format(r))
+    else:
+        print('your guess ({}) is too big'.format(r))
+
+########################### End of Syntax exercise ############################

```

Comment:
Please use ```while True``` for the infinite loop. Also could you please use more descriptive variable names?